lkmpg.tex

```diff
@@ -1669,6 +1673,11 @@ \subsection{Bottom Half}
 
 \samplec{examples/bottomhalf.c}
 
+It is expected that at some unspecified point in the future tasklets may eventually be deprecated, to be replaced by \textit{threaded irqs} instead.
```

Comment:
In 2020, the removal of the tasklet API also has been discussed.
See https://lwn.net/Articles/830964/.

Reply:
Also, the patch of tasklets in 2020 has the same improvement as the `timer_setup`. In other words, the CET needs to be mention too.
See:
* https://github.com/torvalds/linux/commit/12cc923f1ccc1df467e046b02a72c2b3b321b6a2
* https://lwn.net/ml/kernel-hardening/20200716030847.1564131-1-keescook@chromium.org/